graphics/src/main/kotlin/ktx/graphics/graphics.kt

```diff
@@ -0,0 +1,93 @@
+package ktx.graphics
+
+import com.badlogic.gdx.graphics.Color
+import com.badlogic.gdx.graphics.glutils.ShapeRenderer
+import com.badlogic.gdx.math.Vector2
+import com.badlogic.gdx.math.Vector3
+
+/** Draws a cube using {@link ShapeType#Line} or {@link ShapeType#Filled}. The pos specify the bottom, left, front corner
+ * of the rectangle. */
+fun ShapeRenderer.box(pos: Vector3, width: Float, height: Float, depth: Float) {
+  this.box(pos.x, pos.y, pos.z, width, height, depth)
+}
+
+/** Calls {@link #circle(float, float, int)} by estimating the number of segments needed for a smooth circle. */
+fun ShapeRenderer.circle(pos: Vector2, radius: Float) {
+  this.circle(pos.x, pos.y, radius)
+}
+
+/** Draws a circle using {@link ShapeType#Line} or {@link ShapeType#Filled}. */
+fun ShapeRenderer.circle(pos: Vector2, radius: Float, segment: Int) {
+  this.circle(pos.x, pos.y, radius, segment)
+}
```

Comment:
Can this and the previous functions be merged with a default argument for `segment`?